src/analysis/child_properties.rs

```diff
@@ -100,7 +100,12 @@ fn analyze_property(
             imports.add_used_types(rust_type.used_types());
         }
 
-        let get_out_ref_mode = RefMode::of(env, typ, library::ParameterDirection::Return);
+        let get_out_ref_mode = RefMode::of(
+            env,
+            typ,
+            library::ParameterDirection::Return,
+            library::Transfer::None,
```

Comment:
Not sure if this is a sensible default.

Reply:
As sensible as any other :) There's no good default here.